flashpay/apps/account/authentication.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any, Dict, Optional, Tuple
+
+from rest_framework_simplejwt.authentication import JWTAuthentication
+from rest_framework_simplejwt.exceptions import InvalidToken
+from rest_framework_simplejwt.settings import api_settings
+
+from rest_framework.authentication import BaseAuthentication
+from rest_framework.exceptions import AuthenticationFailed
+from rest_framework.request import Request
+
+from flashpay.apps.account.models import Account, APIKey
+
+
+class PublicKeyAuthentication(BaseAuthentication):
+    def authenticate(self, request: Request) -> Optional[Tuple[Any, Any]]:
```

Comment:
Looking at the function logic, the tuple contents aren't any right? Why use `Any, Any` idolo?

Reply:
That's what my vscode extension gave and I didn't bother to edit it as I thought that was the signature of the base class `authenticate` method. I'd update it